### ISSUE TITLE
Support tuning to a validation dataset (instead of cross-validation)

### ISSUE THREAD

**Issue by kylegilde** [tidymodels/tune#355]
I was wondering if any consideration has been given to adding an option to the tune functions to use a single validation dataset instead of performing cross-validation. 

Here are my reasons for preferring the use of a validation dataset over cross-validation:

1. My dataset contains several years of customer orders. However, I primarily want the model to fit well on the most recent couple of months of orders. Hence, I use a validation dataset containing the most recent months in order to tune my hyperparameters. (I also use a separate test dataset for final evaluation)

2. I have too many observations & features and not enough computing power to perform cross-validation in a reasonable amount of time.

Thank you!

**Comment by juliasilge** [comment]
Yes! We demonstrate how to tune using a validation set [in this article on `tidymodels.org`](https://www.tidymodels.org/start/case-study/), and discuss briefly [in this section](https://www.tmwr.org/resampling.html#validation) how to think about a validation set as a single iteration of resampling.

**Comment by kylegilde** [comment]
Thank you for pointing me to validation_split. Is there a way for me to specify that I want the validation set to contain the most recent 60 days of observations?

**Comment by juliasilge** [comment]
Ah, you may instead want to create a custom resampling object, if you have very specific needs. You can check out how to do this [via `manual_rset()`](https://rsample.tidymodels.org/reference/manual_rset.html) and `make_splits()`. Also check out:

- [this Stack Overflow answer](https://stackoverflow.com/a/64004636/5468471)
- [this GitHub issue](https://github.com/tidymodels/rsample/issues/158)

for some examples on how to create custom resampling objects.

**Comment by kylegilde** [comment]
Thank you!
